debian/watch

```diff
@@ -0,0 +1,2 @@
+# The project has no releases, yet, so I am incrementing the patch
+# version manually and using things directly from git.
```

Comment:
I guess you could count the number of commits as a stable increasing version component and for the `watch` purposes.